hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -41,6 +41,8 @@
       "dfs.container.ipc";
   public static final int DFS_CONTAINER_IPC_PORT_DEFAULT = 9859;
 
+  public static final String OZONE_METADATA_DIRS="ozone.metadata.dirs";
```

Comment:
NIT: space

Reply:
Addressed review comment. Thanks for checking @bharatviswa504 .